framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/NuGet/NuGetService.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Volo.Abp.Json;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Threading;
+
+namespace Volo.Abp.Cli.NuGet
+{
+    public class NuGetService : ISingletonDependency
+    {
+        protected IJsonSerializer JsonSerializer { get; }
+        protected ICancellationTokenProvider CancellationTokenProvider { get; }
+
+
+        public NuGetService(
+            IJsonSerializer jsonSerializer,
+            ICancellationTokenProvider cancellationTokenProvider)
+        {
+            JsonSerializer = jsonSerializer;
+            CancellationTokenProvider = cancellationTokenProvider;
+        }
+
+        public async Task<string> GetLatestVersionAsync(string packageId)
+        {
+            using (var client = new HttpClient())
+            {
+                client.Timeout = TimeSpan.FromSeconds(30);
+                
+                var responseMessage = await client.GetAsync(
+                    $"https://api.nuget.org/v3-flatcontainer/{packageId.ToLowerInvariant()}/index.json",
```

Comment:
This endpoint seems returning all versions including previews/betas. Is there a way of filtering only stable versions?

Reply:
Not filter by endpoint but i examined to nuget client is using SemanticVersion parser. Parsing version after now is prerelease here i implement this my code.
[https://github.com/NuGet/NuGet.Client/blob/3803820961f4d61c06d07b179dab1d0439ec0d91/src/NuGet.Core/NuGet.Versioning/SemanticVersion.cs#L216-L233](url)

Reply:
OK, then we can do it in a similar way: Check if version contains `beta`, `preview` or `alpha` :)
It would be good if you update PR, otherwise I can handle it after merge.
It is good to add a parameter like `bool includePreviews=false` to the ``GetLatestVersionAsync`` method. Thanks.

Reply:
OK i handle it :)

Reply:
Thanks a lot :)